text/0000-source-date-epoch.md

```diff
@@ -0,0 +1,84 @@
+# Meta
+[meta]: #meta
+- Name: Use SOURCE_DATE_EPOCH to configure image create time
+- Start Date: 2022-02-24
+- Author(s): natalieparellano
+- Status: Draft
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+To allow for [build reproducibility](https://github.com/buildpacks/spec/blob/main/platform.md#build-reproducibility), images created by Cloud Native Buildpacks have a hard-coded create time of January 1, 1980. We have received requests from the community (see [here](https://github.com/buildpacks/pack/issues/1281) and [here](https://buildpacks.slack.com/archives/C94UJCNV6/p1643842965830459)) to allow for this value to be configurable. This RFC proposes making the value configurable by setting `SOURCE_DATE_EPOCH` in the lifecycle's execution environment during `export`.
```

Comment:
I dislike that this environment variable doesn't align with the `CNB_` prefix of all other environment variables. I get that it comes from a [convention](https://reproducible-builds.org/specs/source-date-epoch/) but I'm not sure that it justifies breaking away from our convention. Even `ko` has `KO_DATA_DATE_EPOCH `. What if we left `SOURCE_DATE_EPOCH` up to the platforms to decide support on but have `CNB_DATE_EPOCH` or something similar for our interface?

Reply:
IIUC `KO_DATA_DATE_EPOCH` is used for a slightly different purpose: "used to set the modtime timestamp of the files in `KO_DATA_PATH`". However jib uses `USE_CURRENT_TIMESTAMP` to set image create time.

Reply:
What about an emoji vote? `SOURCE_DATE_EPOCH` ❤️  - `CNB_ SOURCE_DATE_EPOCH` 🎉